src/components/organisms/CompareModal/ResourceSetSelector.tsx

```diff
@@ -75,3 +68,154 @@ export const ResourceSetSelector: React.FC<Props> = ({side}: Props) => {
     </S.ResourceSetSelectorDiv>
   );
 };
+
+function ResourceSetTypeSelect({side}: Props) {
```

Comment:
Shall I extract all these selectors to separate files? At that point I feel like the ResourceSetSelector should become its own directory.

Reply:
yeah, I think that makes sense